haskell/cabal.bzl

```diff
@@ -72,7 +72,14 @@ main = defaultMain
         )
     return setup
 
-def _prepare_cabal_inputs(hs, cc, dep_info, cc_info, cabal, setup, srcs, cabal_wrapper_tpl, package_database):
+_CABAL_TOOLS = ["alex", "c2hs", "cpphs", "doctest", "happy", "tar"]
```

Comment:
Which `tar` is that. Is it the binary `tar`, or the Hackage package [`tar`](http://hackage.haskell.org/package/tar)?

Reply:
Removed.